jujugui/static/gui/src/app/components/deployment/summary/summary.js

```diff
@@ -122,10 +171,7 @@ YUI.add('deployment-summary', function() {
         return;
       }
       // Validate both fields if we are creating a new model
-      if (!this.props.validateForm([
-        'modelName',
-        'templateRegion'
-      ], this.refs)) { return; }
+      if (!this.props.validateForm(['modelName'], this.refs)) { return; }
```

Comment:
Do we not need to validate that one of the options was selected (I couldn't QA this to see what exactly happens here).